api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -159,7 +160,7 @@ async function balanceV2Reducer(
     type: 'STAKE', // just a staking denom on Kusama for now
     available: availableLunieCoin.amount,
     total: totalLunieCoin.amount,
-    staked: stakedLunieCoin.amount,
+    staked: stakedLunieCoin.amount - undelegating,
```

Comment:
oh woah, how could we miss that

Reply:
mmm, actually you just need to use active, no? It is the same

Reply:
For example:

```
"stash": "5GNtHmp1XmHJkGnM7swchrNGDcJ3r9yaQkumDZYK8tuEeZjG",
"total": 118362691275,
"active": 117362691275,
"unlocking": [
{
"value": 1000000000,
"era": 2717
}
],
```

Reply:
We could also include unstaking in balanceV2 actually :shrug: It is not a bad idea

Reply:
alright! let's use active. didn't see that. do we need to query for `unstaking` anywhere?